tpot/base.py

```diff
@@ -263,6 +265,7 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
         self.verbosity = verbosity
         self.disable_update_check = disable_update_check
         self.random_state = random_state
+        self._file = log_file
```

Comment:
Based on [scikit-learn API for BaseEstimator](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html#sklearn.base.BaseEstimator), the line should be self.log_file=log_file. Then TPOT object can both use `get_params` and `set_params`.

Reply:
That makes a lot sense. It was finished. Thanks.